Stage 4 sometimes reporting start_background_mysql_upgrade failure when command returns successfully

The following failure is sometimes observed during stage 4:

```
=> Log opened from cPanel Update (upcp) - Slave (32322) at Mon Apr  4 19:48:27 2022
[2022-04-04 19:48:27 +0000]   Pre Maintenance completed successfully
[2022-04-04 19:48:27 +0000]   95% complete
[2022-04-04 19:48:27 +0000]   Running Standardized hooks
[2022-04-04 19:48:27 +0000]   100% complete
[2022-04-04 19:48:27 +0000]
[2022-04-04 19:48:27 +0000] 	cPanel update completed
[2022-04-04 19:48:27 +0000]   A log of this update is available at /var/cpanel/updatelogs/update.32322.12140.1649101515.log
[2022-04-04 19:48:27 +0000]   Removing upcp pidfile
[2022-04-04 19:48:27 +0000]
[2022-04-04 19:48:27 +0000] Completed all updates
=> Log closed Mon Apr  4 19:48:27 2022

* 04-19:48:28 (3115) [INFO] Running: /usr/bin/rm -f /usr/local/cpanel/cpsanitycheck.so


* 04-19:48:28 (3115) [INFO] Running: /usr/local/cpanel/cpkeyclt

Updating cPanel license...Done. Update succeeded.

* 04-19:48:29 (1621) [INFO] Removing leapp from excludes in /etc/yum.conf
* 04-19:48:29 (1506) [INFO] Restoring MySQL 8.0
* 04-19:48:39 (1521) [INFO] Restoring MySQL via upgrade_id mysql_upgrade.20220404-194839
* 04-19:48:39 (1522) [INFO] Waiting for MySQL installation
..........
..
* 04-19:49:41 (1552) [FATAL] Failed to restore MySQL 8.0: upgrade mysql_upgrade.20220404-194839 status 'failed'
* 04-19:49:41 (1553) [FATAL] ---
data:
  upgrade_id: mysql_upgrade.20220404-194839
metadata:
  command: start_background_mysql_upgrade
  reason: OK
  result: 1
  version: 1
* 04-19:49:42 (489) [ERROR] Sending notification: Fail to update to AlmaLinux 8
* 04-19:49:42 (490) [ERROR] The elevation process failed during stage 4.

You can continue the process after fixing the errors by running:

    /usr/local/cpanel/scripts/elevate-cpanel --continue

You can check the error log by running:

    /usr/local/cpanel/scripts/elevate-cpanel

Last Error:

Failed to restore MySQL at /usr/local/cpanel/scripts/elevate-cpanel line 1554.


* 04-19:49:42 (391) [FATAL] Failed to restore MySQL at /usr/local/cpanel/scripts/elevate-cpanel line 1554.
Failed to restore MySQL at /usr/local/cpanel/scripts/elevate-cpanel line 1554.

^C
```

The WHM API call appears to be successful, but the script is failing to see it as such. Restarting with `--continue` usually succeeds without issue.

--- a/elevate-cpanel
+++ b/elevate-cpanel
@@ -1604,8 +1604,8 @@ sub reinstall_mysql_packages {
         my $c = 0;
 
         while (1) {
-            $c = ( $c + 1 ) % 10;
-            my $out = Cpanel::SafeRun::Simple::saferunnoerror( qw{/usr/local/cpanel/bin/whmapi1 background_mysql_upgrade_status }, "upgrade_id=$id" );
+            $c   = ( $c + 1 ) % 10;
+            $out = Cpanel::SafeRun::Simple::saferunnoerror( qw{/usr/local/cpanel/bin/whmapi1 background_mysql_upgrade_status }, "upgrade_id=$id" );
             die qq[Fail to restore MySQL $mysql_version: cannot check upgrade_id=$id] if $?;
 
             if ( $out =~ m{\sstate:\s*inprogress} ) {

